### PR TITLE
🐞Fix: Github Ticket button in MobileView of Bounty Description

### LINKED ISSUE #1161: Add Github Ticket button in MobileView of Bounty Description
**Describe the bug**

The bounty body should have a href button to link to the github issue but in mobile view the button does not exist.

https://github.com/stakwork/sphinx-tribes/assets/91251535/cf0fd09a-2344-4322-bcdd-d7099eff4b5c

**Expected behavior**
A button to be added at the very top which links to the github issue which is referenced in the specific bounty

**Smartphone (please complete the following information):**
 - Device: iPhone 12Mini
 - OS: iOS 17.2
 - Browser -Safari

### Acceptance Criteria
- [ ] I have tested on Chrome desktop
- [ ] I have posted a screenshot or video if it is a change in the front end

### PR DESCRIPTION
**Pull request by VividhPandey003**
## Describe your changes

1. fixed props of github button
2. added share to twitter for mobile view
3. removed the mark unpaid button

## Issue ticket number and link
Closes  #1161 

## Type of change

https://github.com/stakwork/sphinx-tribes/assets/91251535/a2cdaad8-aa0b-4cae-9cbe-9c7c9a093c30

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [x] I have tested on a mobile device
- [x] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
LGTM. Let's get a code review

File: frontend/app/src/people/widgetViews/summaries/wantedSummaries/CodingMobile.tsx [hunk 1]
```diff
@@ -152,11 +153,14 @@ export default function MobileView(props: CodingViewProps) {
...
-            <ShareOnTwitter {...props} />
+            <ShareOnTwitter
+              issueCreated={created}
+              ownerPubkey={owner_id}
+              labels={labels}
+              titleString={titleString}
+            />
           </ButtonRow>
 
-          {markUnpaid}
```
Comment (kevkevinpal) [inline frontend/app/src/people/widgetViews/summaries/wantedSummaries/CodingMobile.tsx:158]: why was this removed? not sure what it does either

Comment (VividhPandey003) [reply frontend/app/src/people/widgetViews/summaries/wantedSummaries/CodingMobile.tsx:158]: oh well if you see the current deployed version in mobile view it has this mark as unpaid button. but this is not present in the desktop view. I tried to copy the desktop view and thus to get the uniformity i have removed this

Comment (VividhPandey003) [reply frontend/app/src/people/widgetViews/summaries/wantedSummaries/CodingMobile.tsx:158]: example 

image

Comment (kevkevinpal) [reply frontend/app/src/people/widgetViews/summaries/wantedSummaries/CodingMobile.tsx:158]: The mark unpaid also exists on desktop

We just should only show the mark unpaid button when the bounty is paid

Comment (VividhPandey003) [reply frontend/app/src/people/widgetViews/summaries/wantedSummaries/CodingMobile.tsx:158]: Okay. Sorry for removing it. I'll fix it right away

Comment (kevkevinpal) [reply frontend/app/src/people/widgetViews/summaries/wantedSummaries/CodingMobile.tsx:158]: I would add `{ !!paid ? markUnpaid : null }` So it only shows when paid

**Comment by ecurrencyhodler** [comment]
Tested and it works. GJ !
